client.tf

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#-------------------#
+# Forseti templates #
+#-------------------#
+data "template_file" "forseti_client_startup_script" {
+  template = "${local.client_startup_script}"
+
+  vars {
+    forseti_environment = "${data.template_file.forseti_client_environment.rendered}"
+    forseti_repo_url    = "${var.forseti_repo_url}"
+    forseti_version     = "${var.forseti_version}"
+    forseti_home        = "${var.forseti_home}"
+  }
+}
+
+data "template_file" "forseti_client_environment" {
+  template = "${local.client_env_script}"
+
+  vars {
+    forseti_home             = "${var.forseti_home}"
+    forseti_client_conf_path = "${local.client_conf_path}"
+  }
+}
+
+data "template_file" "forseti_client_config" {
+  template = "${local.client_conf}"
+
+  vars {
+    forseti_server_ip = "${google_compute_instance.forseti-server.network_interface.0.access_config.0.assigned_nat_ip}"
+  }
+}
+
+resource "google_compute_instance" "forseti-client" {
+  name                      = "${local.client_name}"
+  zone                      = "${local.client_zone}"
+  project                   = "${var.project_id}"
+  machine_type              = "${var.client_type}"
+  allow_stopping_for_update = true
+
+  boot_disk {
+    initialize_params {
+      image = "${var.client_boot_image}"
+    }
+  }
+
+  network_interface {
+    subnetwork_project = "${var.project_id}"
```

Comment:
Looks like this is missing the `vpc_host_subnetwork` variable?
`"${var.vpc_host_subnetwork == "" ? var.project_id : var.vpc_host_subnetwork}"`